src/main/java/uk/gov/pay/directdebit/payments/api/CollectPaymentRequestValidator.java

```diff
@@ -13,7 +13,7 @@
     private final static String REFERENCE_KEY = "reference";
     private final static String MANDATE_ID_KEY = "mandate_id";
 
-    private final static int MIN_AMOUNT_IN_PENCE = 1;
+    private final static int MIN_AMOUNT_IN_PENCE = 100;
```

Comment:
I suppose really this should be `1_00` for consistency with `MAX_AMOUNT_IN_PENCE`